data/global/excel/uniqueitems.txt

```diff
@@ -1420,4 +1420,4 @@ t5 Splash Charm	1415	100	0			1	1	1	30	cm4	charm	1	1	1			flpmss	invmss	item_gem	1
 t6 Splash Charm	1416	100	0			1	1	1	45	cm4	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashthree	proc_SplashDamage_Three	100	1	dmg2%		-20	-27																																										0
 t7 Splash Charm	1417	100	0			1	1	1	60	cm4	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashthree	proc_SplashDamage_Three	100	1	dmg2%		-15	-25																																										0
 t8 Splash Charm	1418	100	0			1	1	1	75	cm4	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashthree	proc_SplashDamage_Three	100	1	dmg2%		-10	-23																																										0
-t9 Splash Charm	1419	100	0			1	1	1	90	cm5	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashfour	proc_SplashDamage_Four	100	1	dmg2%		-5	-20																																										0
+t9 Splash Charm	1419	100	1			1	1	1	90	cm5	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashfour	proc_SplashDamage_Four	100	1	dmg2%		-5	-20																																										0
```

Comment:
This may unlock dropping this directly. We may want to make the level requirement super high there may be a field that is like "droppable" toggle that off if it's on. 

May not be a problem though just something I wanted to point out.

Reply:
thanks for mentioning it !